tests/t566-operator.c

```diff
@@ -0,0 +1,184 @@
+/// @file
+/// Test assembly of non-symmetric mass matrix operator (multi-component) see t537
+/// \test Test assembly of non-symmetric mass matrix operator (multi-component)
+#include <ceed.h>
+#include <stdlib.h>
+#include <math.h>
+#include "t566-operator.h"
+
+int main(int argc, char **argv) {
+  Ceed ceed;
+  CeedElemRestriction elem_restr_x, elem_restr_u,
+                      elem_restr_qd_i;
+  CeedBasis basis_x, basis_u;
+  CeedQFunction qf_setup, qf_mass;
+  CeedOperator op_setup, op_mass;
+  CeedVector q_data, X, U, V;
+  CeedInt P = 3, Q = 3, dim = 2, num_comp = 2;
+  CeedInt n_x = 1, n_y = 1;
+  CeedInt num_elem = n_x * n_y;
+  CeedInt num_dofs = (n_x*2+1)*(n_y*2+1), num_qpts = num_elem*Q*Q;
```

Comment:
I think this is intended. I'd like to be able to simplify as much as possible.
```suggestion
  CeedInt num_dofs = (n_x*(P-1)+1)*(n_y*(P-1)+1), num_qpts = num_elem*Q*Q;
```

Reply:
BTW, I think "dofs" used here to mean number of nodes (not `num_dofs = num_nodes * num_comp`) is confusing.